django_downloadview/files.py

```diff
@@ -239,7 +241,12 @@ def request(self):
 
     @property
     def file(self):
-        return self.request.raw
```

Comment:
This caused the issue of #97.
Replaced by a combination of `request.iter_content()` and `BytesIteratorIO` so that file is not fully loaded into memory.